src/Graphing/Enrich.hs

```diff
@@ -0,0 +1,230 @@
+module Graphing.Enrich (
```

Comment:
I think this should be called something like `Graphing.EnrichConan`. `Graphing.Enrich` seems too generic, considering that all of these functions are specific to Conan.